batch7crud-roll8/src/main/webapp/WEB-INF/views/employees/edit.jsp

```diff
@@ -0,0 +1,45 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Grishma Shrestha <grishmashrestha@lftechnology.com>
+  Date: 1/20/16
+  Time: 7:34 AM
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<!DOCTYPE html>
+<html>
+<head>
+  <base href="${pageContext.request.contextPath}/">
+  <title>Edit Details of Employee</title>
+</head>
+<body>
+<div>
+  <h1>Edit Employee Form</h1>
+  <div>
+    <form action="employees/${employee.getId()}/edit" method="POST">
+      <div>
+        <label>Name:</label>
```

Comment:
Just for heads up, how you can internationalize this application? ie. if you have to localize this application in French language what are the steps to support it.